F1/F1/Standings/DriverStandingTableViewCell.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  AllResultsTableViewCell.swift
+//  F1
+//
+//  Created by Kaitlyn Sookdhew on 2024/04/08.
+//
+
+import UIKit
+
+class DriverStandingTableViewCell: UITableViewCell {
```

Comment:
This looks similar to the other cell. In future, if you want to try, see if you can use the saem cell and make it generic to limit the same code being dome twice :)

Reply:
So I did consider this in the beginning but there are a few differences and I decided not the sake of the way I did it was quicker. I am definitely keeping this in mind for improvements and will implement after I have done everything else 😄